app/components/shared/DropdownMenu.vue

```diff
@@ -86,6 +86,7 @@
   border: 0 !important;
   padding: 8px !important;
   color: inherit !important;
+  background-color: var(--background) !important;
```

Comment:
I assume you can get rid of all `!important` here if you specify selector `.popper.dropdown-menu`